src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -9201,6 +9201,7 @@ and TcMethodApplicationThen
        m           // The range of the object argument or whole application. We immediately union this with the range of the arguments
        mItem       // The range of the item that resolved to the method name
        methodName // string, name of the method
+       (methodNameRange: range option)
```

Comment:
Why not use/fix `mItem` range?

Reply:
Thanks for the review. I Agree fixing `mItem` range seems a proper fix. Will have a look

Reply:
The original reason (at least from my end) was to add a new range as it was a bit of a guess on what would happen when we update the existing `m`. I couldn't really asses the impact so it felt safer to just start with a very specific fix and then expand on that.